src/app/child-dev-project/aser/model/aser.spec.ts

```diff
@@ -77,15 +79,19 @@ describe("Aser", () => {
     const id = "test1";
     const entity = new Aser(id);
 
-    expect(entity.getWarningLevel()).toBe(WarningLevel.OK);
+    expect(entity.getWarningLevel()).toBe(
+      warningLevels.find((level) => level.id === "OK")
+    );
   });
 
   it("warning level WARNING if some bad results", function () {
     const id = "test1";
     const entity = new Aser(id);
-    entity.english = Aser.ReadingLevels[0];
-    entity.math = Aser.MathLevels[1];
+    entity.english = readingLevels[1];
+    entity.math = readingLevels[2];
 
-    expect(entity.getWarningLevel()).toBe(WarningLevel.WARNING);
+    expect(entity.getWarningLevel()).toBe(
+      warningLevels.find((level) => level.id === "WARNING")
```

Comment:
As the Warning Level is quite deeply connected to business logic in code, maybe we should wrap this into its own object/enum rather than relying on string filtering